lib/tmfUtils.js

```diff
@@ -296,6 +321,44 @@ exports.isValidStatusTransition = function(firstSt, nextSt) {
     return firstInd > -1 && nextInd > -1 && (nextInd == firstInd + 1 || nextInd == firstInd)
 }
 
+exports.isValidPrice = function(price, unit) {
+    if (typeof price !== 'number') {
+        return false
+    }
+    if (price < 0.01 || price > 1000000000) {
+        return false
+    }
+    const digits = currencyCodes.code(unit).digits;
+    // check if price has 2 digits after the decimal point
+    const priceStr = price.toString()
+    const priceParts = priceStr.split('.')
+    if (priceParts.length > 1 && priceParts[1].length > digits) {
+        return false
+    }
+    return true
+}
+
+exports.isValidPercentage = function(percentage) {
+    if (typeof percentage !== 'number') {
```

Comment:
This is not true the API supports numbers to be provided in string format and is correct